packages/office-ui-fabric-react/src/components/TextField/TextField.styles.tsx

```diff
@@ -153,12 +153,17 @@ export function getStyles(props: ITextFieldStyleProps): ITextFieldStyles {
             }
           }
         },
-        focused &&
+        focused && [
+          {
+            position: 'relative'
+          },
           getInputFocusStyle(
             !hasErrorMessage ? semanticColors.inputFocusBorderAlt : semanticColors.errorText,
             effects.roundedCorner2,
-            'borderBottom'
+            'borderBottom',
+            -2
```

Comment:
should not need to pass & set to `-2` here? (`-1` should be correct)

Reply:
Since we set `borderRadius` to `effects.roundedCorner2`, I assumed we want to see the `borderRadius`. If I remove the param (or set it to the default -1) we are loosing it since the pseudo element `:after` overlays the border set by `.ms-TextField-wrapper`.

#### Please see example with border color set to light green to see the overlaps:
![image](https://user-images.githubusercontent.com/56690996/72599680-78606800-3912-11ea-815b-cad9da9e1b16.png)

#### borderPosition set to -1 or undefined
![image](https://user-images.githubusercontent.com/56690996/72599792-a8a80680-3912-11ea-85ea-121d5a344526.png)

#### borderPosition set to -2
![image](https://user-images.githubusercontent.com/56690996/72599824-b3fb3200-3912-11ea-87b5-919454da9a40.png)

Reply:
@betrue-final-final could you confirm which behavior is expected? should we have `border-radius` if there is only border bottom?

Reply:
Nvm Ben. I checked the [design doc](https://www.figma.com/file/BAfsTrZ9ehIYZ0oYuyskUN2R/Web-Fluent?node-id=0%3A2741). it should be no radius:
![image](https://user-images.githubusercontent.com/1207059/72647983-b9667580-392e-11ea-81c6-030b6ba1aa65.png)

Reply:
@v-jajach - please update the PR accordingly ^